### DISCUSSION TITLE
[Resolved] Ongoing incident

### DISCUSSION THREAD

**Discussion by helinanever** [codemagic-ci-cd/codemagic-docs#2166] [Announcements]
There is an ongoing incident that is causing builds to fail unexpectedly. We are investigating the root cause of the issue and working hard to restore the service. An update will be provided as soon as there is more information available.

**Comment by helinanever** [comment]
Everything is back to normal now and we expect all builds to run as expected.

**Reply by toonleijtens** [reply]
Hi @helinanever, we were having the same problems. The environments are being prepared, and then the process fails (time-out). As we are on pay as you go we are probably charged for the inproductive time spend. It is great to see that you think the problem is addressed and that you expect all builds to be running as expected. But would you charge us? and if so could we be compensated?

**Reply by helinanever** [reply]
Hi @toonleijtens, the issue you are describing is probably not related to this incident but we are willing to credit your account. Please contact the support team through the in-app chat.

**Comment by helinanever** [comment]
Some more details on the incident:

The issue was caused by a deployment that accidentally reverted an earlier change made to our build infrastructure. The incident affected the output of build artifacts, due to which builds failed with “Something went wrong” during the publishing step.

Time window: from 09:07 to 10:02 UTC.

We apologize for any inconvenience this may have caused.
